ast/compile.go

```diff
@@ -536,6 +538,38 @@ func (c *Compiler) GetRulesWithPrefix(ref Ref) (rules []*Rule) {
 	return rules
 }
 
+func (c *Compiler) GetPackageAnnotations(pkg *Package) *Annotations {
+	as := c.annotationSet
+
+	result := make([]*Annotations, 0, 2)
```

Comment:
❓ What's the matter with the 2 hint here? Do we know it's at least two, but it could be more than two?

Reply:
We set the size of the underlying array to 2, guessing/hoping that we don't get more than that, so that `append` doesn't grow the slice beyond what's necessary. Might be a bit excessive for such small a collection ..
But I see now that I can rearrange this to avoid the slice growing at all. Will fix.